### DISCUSSION TITLE
logging in using two factor authentication

### DISCUSSION THREAD

**Discussion by aleith** [community/community#37163] [Other Feature Feedback, Questions, & Ideas] [answered]
i'm trying to login on my desktop using browser and it asks for two factor autho 6-digit code. 

i have an autho app but github isn't listed. 

i forget where my live codes are. what the fuck do i do? 

turns out they are in 1Password, my password program but theres an impossibility of moving forward it and when i gorget that like i did this morning. 

accessibility issue for people with ADHD, and poor memories and/or get overwhelmed. it should prompt me by saying you created the 6-digit two factor autho in you 1password application. or at least allow me to receive an email on registered email address to say the same.

**Comment by ossanna16** [comment, marked as answer]
Hi @aleith,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on 1Password. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by aleith** [reply]
Please note that this isn't a 1password issue. This is an issue about the level of usefulness of the GitHub prompt about finding your six digit code. I often open Authy and Authenticator on my phone to look for it when in fact it's part of setting it up in 1password. It would be useful if GitHub was able to ( if I set it up in my preferences) prompt me where I look for the code or at least provide hints to me as to where I might have this functionality on my phone or desktop Mac.

**Comment by aleith** [comment]
Thanks Anna :-)

Best
Alastair

[Retracted due to PII]

**Reply by queenofcorgis** [reply]
Hi @aleith,

I've removed your private information from your comment as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) 😄
